### ISSUE TITLE
test LAB colour space conversion

### ISSUE THREAD

**Issue by harrisonpim** [wellcomecollection/catalogue#919]

**Comment by harrisonpim** [comment]
LAB space conversion _does_ make a difference to the results we get, but:
- they're perceptually very close (especially when used as filters)
- the proportion of 'wrong' or 'unexpected' results remains very similar
- the perceivable difference in quality of colour similarity queries is also very small
- the distribution of colours between colour-space-bins is similarly extreme in both spaces. 

It seems like [the addition of a 2-bin](https://github.com/wellcomecollection/catalogue/pull/913) has done enough to improve our colour filter results already. If we're going to look at this further, it might be worth going down the voronoi-bins-in-lab-space route, but at this point it's not worth the effort.

**TL;DR** RGB space is sufficient for now, not worth making the change to LAB space.
